clients/runtime/src/cli.rs

```diff
@@ -57,8 +58,18 @@ fn get_credentials_from_file(file_path: &str, keyname: &str) -> Result<Pair, Key
     Ok(pair)
 }
 
-pub fn parse_account_keyring(src: &str) -> Result<AccountKeyring, Error> {
-    AccountKeyring::from_str(src).map_err(|_| Error::KeyringAccountParsingError)
+pub fn parse_account_keyring(src: &str) -> Result<Ed25519Keyring, Error> {
```

Comment:
Unfortunately, `Ed5519Keyring` does not have a `FromStr` implementation (but `Sr25519Keyring` does – strange). I just copied a version here.